jwt/api_jws.py

```diff
@@ -128,7 +128,7 @@ def encode(
     def decode_complete(
         self,
         jwt: str,
-        key: str = "",
+        key: Union[str, bytes] = "",
```

Comment:
I think if bytes is the preferred type, this should default to that. Same for the functions.

```suggestion
        key: Union[str, bytes] = b"",
```